recipes/chaospy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "2.2.4" %}
+
+package:
+  name: chaospy
+  version: {{ version }}
+
+source:
+  fn: chaospy-{{ version }}.tar.gz
+  url: https://github.com/jonathf/chaospy/archive/v{{ version }}.tar.gz
+  sha256: b8c198d9e1139de494dcb1449945d25635721f3cbdd0ca27588ea6000ae4d9ed
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - networkx
+    - scikit-learn
```

Comment:
The `setup.py` also explicitly lists `numpy` and `scipy` as dependencies; they're of course transitively required by `scikit-learn`, but could probably list them for completeness....